benchmark.cfg.example

```diff
@@ -20,7 +20,7 @@ query_levels=[1, 5,10,15,20]
 threads=8
 mode=benchmark
 os=linux
-server_host=127.0.0.1
+server_host=TFB-server
```

Comment:
I see this being done in travis and vagrant, but I don't see this setup happening in the general case (like our production code): you should plan on adding that in.